src/ui/public/vis/editors/default/agg_groups.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const aggGroupNameMaps = () => ({
+  metrics: i18n.translate('common.ui.vis.editors.aggGroups.metricsText', { defaultMessage: 'metrics' }),
```

Comment:
I'm not sure we need to translate it. Let us ask Tim when he will be reviewing it

Reply:
I added translations for it on purpose since it's displayed on Editor

![image](https://user-images.githubusercontent.com/11524637/46671105-96ea9600-cbdc-11e8-9f52-ebd1ea71407a.png)

Reply:
Makes sense for me to have those translated.